.github/workflows/spanner-staging-tests.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     name: Spanner Dataflow Templates Integration Tests
     timeout-minutes: 180
     # Run on any runner that matches all the specified runs-on values.
-    runs-on: [ self-hosted, it ]
+    runs-on: [ self-hosted, spanner ]
```

Comment:
earlier the parameters were it-project etc - corresponding to it.

Now it says Spanner. Where does the name get used ?

Reply:
This is the tag of github actions runner on which needs to pick up this workflow. I have created a new pool of github actions runners and tagged it `spanner` . These are in new project and is done to provide isolation for the staging tests